src/frontend/apps/desk/src/components/Link.tsx

```diff
@@ -3,9 +3,12 @@ import styled from 'styled-components';
 
 export interface LinkProps {
   $css?: string;
+  tabIndex?: 0 | -1;
 }
 
-export const StyledLink = styled(Link)<LinkProps>`
+export const StyledLink = styled(Link).attrs<LinkProps>(({ tabIndex = 0 }) => ({
```

Comment:
I don't think you need all these changes. `Link` is naturally focusable because it is a wrapper of `<a>` tag.